### PR TITLE
add  support for override fields

### PR DESCRIPTION
**Pull request by yding03**
This PR will add support for override fields.

### CODE REVIEW

**Review by benbeltzer — APPROVED** [review]
lgtm

**Review by jnnnthnn — APPROVED** [review]
Looks great! I also tested this locally and can confirm it works as expected.

File: README.md [hunk 1]
```diff
@@ -106,6 +106,7 @@ Parameters:
 
 - `responsePayload: string` - A string describing the payload type to return when fetching transaction metadata, e.g. "us-dl". See [our testing guide](https://docs.berbix.com/docs/testing) for possible options.
 - `flags: string[]` - An optional list of flags to associate with the transaction (independent of the payload's contents), e.g. ["id_under_18", "id_under_21"]. See [our flags documentation](https://docs.berbix.com/docs/id-flags) for a list of flags.
+- `override_fields: array(string => string)` - An optional mapping from a [transaction field](https://docs.berbix.com/reference#gettransactionmetadata) to the desired override value, e.g. `'override_fields' => array('date_of_birth' => '2000-12-09')`
```
Comment (jnnnthnn) [inline README.md]: Nit: this should be camelCase for consistency.
```suggestion
- `overrideFields: array(string => string)` - An optional mapping from a [transaction field](https://docs.berbix.com/reference#gettransactionmetadata) to the desired override value, e.g. `'overrideFields' => array('date_of_birth' => '2000-12-09')`
```

File: examples/transaction.php [hunk 2]
```diff
@@ -27,6 +27,7 @@
 $transaction = $client->overrideTransaction($tokens, array(
   'responsePayload' => 'us-dl',
   'flags' => array('id_under_18', 'id_under_21'),
+  'override_fields' => array('date_of_birth' => '2000-12-09'),
```
Comment (jnnnthnn) [inline examples/transaction.php]:
```suggestion
  'overrideFields' => array('date_of_birth' => '2000-12-09'),
```

File: lib/Client.php [hunk 3]
```diff
@@ -144,6 +144,9 @@ public function overrideTransaction($tokens, $params) {
     if (array_key_exists('flags', $params)) {
       $payload['flags'] = $params['flags'];
     }
+    if (array_key_exists('override_fields', $params)) {
+      $payload['override_fields'] = $params['override_fields'];
+    }
```
Comment (jnnnthnn) [inline lib/Client.php]:
```suggestion
    if (array_key_exists('overrideFields', $params)) {
      $payload['override_fields'] = $params['overrideFields'];
    }
```

**Review by jnnnthnn — COMMENTED** [review]
A few minor nits to ensure consistency.

**Pull request merged**